app/controllers/api/v1/admin/organizations_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V1
+    module Admin
+      class OrganizationsController < ApiController
+        include Api::Admin::OrganizationsController
```

Comment:
Since the V0 API is deprecated and going to be removed, we don't really gain from continuing the concerns/include pattern. In [other](https://github.com/forem/forem/pull/19542) [cases](https://github.com/forem/forem/pull/19009), we've tended to prefer "inlining" the controller actions. I _think_ the intent is that this pattern helps with maintaining v0/v1 parity, but isn't useful/preferable when we're adding new features only in one place.